src/bin/pgcopydb/table-data.c

```diff
@@ -1196,12 +1250,11 @@ copydb_table_create_lockfile(CopyDataSpec *specs,
 
 
 /*
- * copydb_mark_table_as_done creates the table doneFile with the expected
- * summary content. To create a doneFile we must acquire the synchronisation
- * semaphore first. The lockFile is also removed here.
+ * copydb_save_copy_progress saves the duration and the number of
+ * bytes transmitted about the copy progress into the catalog.
  */
 bool
-copydb_mark_table_as_done(CopyDataSpec *specs,
+copydb_save_copy_progress(CopyDataSpec *specs,
 						  CopyTableDataSpec *tableSpecs)
 {
 	DatabaseCatalog *sourceDB = &(specs->catalogs.source);
```

Comment:
I do not understand this change. The `summary_finish_table` function is responsible to maintain the `done_time_epoch` field in the SQLite catalogs, and the rest of the code uses that information to determine if the table COPY is finished or not. Now we will update that `done_time_epoch` field multiple times during the COPY. What am I missing?

Reply:
pushed a change for this.